core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/AzureBatchSource.java

```diff
@@ -61,7 +61,6 @@ public AzureBatchSource(AzureBatchConfig config) {
     @Override
     protected Map<String, String> getFileSystemProperties() {
       Map<String, String> properties = new HashMap<>(super.getFileSystemProperties());
-      properties.put("fs.defaultFS", String.format("wasb://%s@%s/", container, account));
```

Comment:
Another option is to prefix the file names with `wasb://<container>@<account>` and only take file names without file system in the config. I guess this would be the better option right @albertshau ?